README.md

```diff
@@ -45,6 +45,14 @@ NumPy requires `pytest` and `hypothesis`.  Tests can then be run after installat
 
     python -c 'import numpy; numpy.test()'
 
+Code of Conduct
+----------------------
+
+NumPy is a community-driven open source project developed by a very diverse group of
```

Comment:
nit: 
```suggestion
NumPy is a community-driven open source project developed by a diverse group of
```

Reply:
This was copied from the website (https://numpy.org/community/). Would you like to make a change there too?

Reply:
I think it would be best - I think it reads better and is more honest :)

Reply:
That reminded of the first post in this [hackernews thread](https://news.ycombinator.com/item?id=29665661):

> I even had to unlearn using the word "very": I would say "very fast" or "very good" or "very whatever else", and they basically said "cut all the very's, just say fast or good or whatever else, and let the reader decide whether it's very".

Reply:
That's an interesting discussion, @WarrenWeckesser! Thank you for sharing!